spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
```

Comment:
- [ ] All your tests are failing on my end. If the tests are passing on your end kindly include a screenshot so that the next reviewers can observe it and would not warrant unnecessary changes.

Reply:
Hi @julie-ify, I find that extremely hard to believe all test cases pass on my end. 
![Tests](https://github.com/tsheporamantso/Blog-App/assets/110445380/c7baf41a-930c-418b-a990-e426f38a9086)

Reply:
Please see another screenshot testing on features
![Screenshot_2](https://github.com/tsheporamantso/Blog-App/assets/110445380/f62e8dcf-7880-4055-876d-18c5f6523813)